reddit_about/pages.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, *args, **kwargs):
         self.help = sections.get('help')
         blurbs = SelfServeBlurb.get_all(return_dict=True)
         if 'platform' in blurbs:
-            formatted_cpm = format_currency(g.cpm_selfserve.decimal, 'USD', locale=c.locale)
+            formatted_cpm = format_currency(g.cpm_selfserve_collection.decimal, 'USD', locale=c.locale)
```

Comment:
Could we do something like `min([g.cpm_selfserve_collection, g.cpm_selfserve, g. cpm_selfserve_geotarget_metro]).decimal`